sim/battle.ts

```diff
@@ -2496,6 +2502,7 @@ export class Battle {
 			}
 		}
 		action.sourceEffect = sourceEffect;
+		action.priority = 7.1;
```

Comment:
Maybe something significantly higher, like 10? Unless there's a reason for this.

Reply:
Also, with my optimization, this doesn't actually need to happen afaict.

Reply:
Oh, nvm, it is move actions being prioritized. Yeah, this is necessary.

Reply:
I set the priority level to 10 in prioritizeAction